content/04.results.md

```diff
@@ -145,52 +147,52 @@ Similarly, we find the same patterns in quoted speakers when comparing against t
 We then linked all the identified region-related named entities to countries with OpenStreetMap.
 Since there may be errors in both coreNLP and OpenStreetMap, we only assumed a country was mentioned when at least two unique entities mapped to the same country in a single article.
-On a benchmark set, this results in a class-balanced error of [XX] across all countries (Supplemental Figure 1 b).
- 
+On a benchmark set, we identify 4 country identifications from a total of 59 country predictions were incorrect (Figure {@fig:suppfig1}b,c).
+When aggregating over articles, we find that 4/30 articles contain exactly one incorrect country mention (Figure {@fig:suppfig1}b)
  
-Once we calculated the mention rate and used the previously described citation rate, we identified countries with a consistent skew towards either a higher or lower mention to citation rate (Figure @fig:fig4 d and Supplemental Figure 4 b).
+Once we calculated the mention rate and used the previously described citation rate, we identified countries with a consistent skew towards either a higher or lower mention to citation rate (Figure {@fig:fig4}d and Figure {@fig:suppfig4}b).
 This is defined as countries where the difference between citation and mention rates is in the top or bottom 5% per year.
 This outlier description allowed us to identify two sets of countries based on their citation and mention rates.
 Those with a high relative citation to mention rate were: Germany, Netherlands, Japan, Spain, Denmark, Israel, Sweden, Switzerland, Belgium, Austria, China, South Korea, and Australia.
 Those with a low relative citation to mention rate: Canada, Democratic Republic of the Congo, Chile, Indonesia, Russia, South Africa, Mexico, Brasil, Philippines, Colombia,  and India.
-We removed all countries that were in both the top or bottom 5% in different years, which excluded the United States and the United Kingdom from consideration. [List all of them]
+We removed all countries that were in both the top or bottom 5% in different years, which excluded the United States, France, and the United Kingdom from consideration.
  
 We then identified content differences between these two sets of countries by analyzing all of the main text from articles that mentioned and did not cite an author affiliated with each of the specified countries.
 After properly identifying high-frequency words across the entire corpus, we identified the top 15 most discriminative terms of each country type ([Methods]).
 Interestingly, we identified that the top 15 words most representative of countries mentioned more were closely related to environmental topics.
-The top 5 terms were rainforest, pesticide, dams, plantations, and deforestation (Figure @fig:fig4 e, Supplemental Figure 4 c and d).
+The top 5 terms were br, pesticide, dams, spores, and herd (Figure {@fig:fig4}e, Figure {@fig:suppfig4}c,d).
```

Comment:
Can we figure out where "br" comes from? It is the only term that is not a word.

Reply:
also "bt" in looking at the figure

Reply:
br comes from unstripped HTML tags. All HTML tags are now stripped. Bt comes from a type of cotton and corn. Updated figure will be added soon.

Reply:
also, just to note, "ES" is from human embryonic stem (ES) cells

Reply:
ok the word ratios are finally updated. The updated figure is here: 
![Screen Shot 2021-06-04 at 11 12 04](https://user-images.githubusercontent.com/1885501/120839037-b4a36100-c525-11eb-8db1-6379c4751f8c.png)

 I did a few tweaks:

1.  Previously I only ran the bootstrap on the top 15 most divergent words by class. After seeing that the bootstraps had high variance, I thought that it would be better to run the bootstraps on all words, then only depict the top 15 per class. This is why some words have disappeared (Bt, Sierra, ES). However, I think the main results are strengthened. We have more agricultural-related words in the "Mention" class.
2. This analysis is now run using a cached word frequency file instead of the raw (copyrighted) files. This is so we ensure the figures are reproducible with the data provided on github.